client/src/pages/chart copy.jsx

```diff
@@ -0,0 +1,235 @@
+import React, { Component } from 'react';
```

Comment:
this file also seems unused.  Pls delete unnecessary files

Reply:
Deleted